### PR TITLE
add container styles, headline for sections styles, section about_us

### PR DESCRIPTION
**Pull request by iryna-kyryllova**

### CODE REVIEW

File: src/css/base.css [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    font-size: 18px;
+    line-height: 1.67;
+  }
+}
+
 h1 {
```
Comment (kharkov-timur) [inline src/css/base.css]: I think we don't need to hardcode this value for h1 tag
Please remove it

File: src/css/base.css [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
 }
 
+body {
+  font-family: var(--primary-font);
+  font-size: 14px;
+  line-height: 1.71;
+  color: rgba(var(--black-rgb), 0.5);
+}
+
+@media screen and (min-width: 768px) {
```
Comment (kharkov-timur) [inline src/css/base.css]: Please combine similar media rules

File: src/css/components/container.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
+.container {
+  margin: 0 auto;
```
Comment (kharkov-timur) [inline src/css/components/container.css]: Where are container max and nim width values?

**Review by kharkov-timur — DISMISSED** [review]

File: src/css/base.css (hunk 2 above)
Comment (iryna-kyryllova) [reply src/css/base.css]: не зрозуміло, де саме... у файлі base.css тільки один media

File: src/css/components/container.css (hunk 3 above)
Comment (iryna-kyryllova) [reply src/css/components/container.css]: min 320px додала, а max є у 1280px. До цього екрану контент не повинен просто тягнутися по ширині?

Comment (kharkov-timur) [reply src/css/components/container.css]: max-width: 1024px;

File: src/css/base.css (hunk 2 above)
Comment (kharkov-timur) [reply src/css/base.css]: Нижче в файлі є ще один @media screen and (min-width: 768px) {

Comment (iryna-kyryllova) [reply src/css/base.css]: Screenshot_7
Не бачу, покажи пліз

Comment (mytrichq) [reply src/css/base.css]: @keira-yes you probably need to merge main to your branch

File: src/css/components/container.css (hunk 3 above)
Comment (mytrichq) [reply src/css/components/container.css]: @keira-yes looks like we already have such styles in base.css

File: src/css/base.css (hunk 2 above)
Comment (iryna-kyryllova) [reply src/css/base.css]: @dimafrontdev I've already done it (please look at my commits)

File: src/css/components/headline.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  content: '';
+  position: absolute;
+  left: 0;
+  top: calc(100% - 14px);
+  width: 100%;
+  height: 12px;
+  border-radius: 8px;
+  background-color: var(--light-green-2);
+  z-index: -1;
+}
```
Comment (mytrichq) [inline src/css/components/headline.css]: duplicated styles from base.css

File: src/css/base.css (hunk 2 above)
Comment (mytrichq) [reply src/css/base.css]: There was merge some minutes ago, sorry

File: src/css/base.css (hunk 1 above)
Comment (iryna-kyryllova) [reply src/css/base.css]: done

File: src/css/base.css (hunk 2 above)
Comment (iryna-kyryllova) [reply src/css/base.css]: done

File: src/css/components/container.css (hunk 3 above)
Comment (iryna-kyryllova) [reply src/css/components/container.css]: updated, done

File: src/css/components/headline.css (hunk 4 above)
Comment (iryna-kyryllova) [reply src/css/components/headline.css]: updated, done

File: src/css/about_us.css [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+  content: '';
+  position: absolute;
+  left: 0;
+  top: calc(100% - 14px);
+  width: 100%;
+  height: 12px;
+  border-radius: 8px;
+  background-color: var(--lighter-green);
+  z-index: -1;
+}
```
Comment (mytrichq) [inline src/css/about_us.css]: Could you please move this to base.css and replace existing solution with yours, thanks ^^)

Comment (iryna-kyryllova) [reply src/css/about_us.css]: done

**Review by mytrichq — APPROVED** [review]

**Comment by mytrichq** [comment]
Please merge main and looks good to go

**Pull request merged**